content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -0,0 +1,207 @@
+---
+reviewers:
+- liggitt
+- mikedanese
+- munnerz
+title: Certificate Signing Requests
+content_template: templates/concept
```

Comment:
This is a concept. In fact, I think the start of this page would have a more natural home in the Concepts section and can then link to the reference documentation to provide more detail. For example, create a new page at https://kubernetes.io/docs/concepts/security/certificate-signing-request/

For an example of what I mean, see https://deploy-preview-19172--kubernetes-io-vnext-staging.netlify.com/docs/concepts/scheduling/kube-scheduler/#kube-scheduler-implementation 
(or, if that PR has merged, see https://kubernetes-io-vnext-staging.netlify.com/docs/concepts/scheduling/kube-scheduler/#kube-scheduler-implementation )